app/controllers/admin/missions_controller.rb

```diff
@@ -18,6 +18,17 @@ def edit; end
 
   def show; end
 
+  def update
+    @mission.java_files.each do |file|
+      target = params[:locations].fetch(file, nil)
+      PiercedLocation.create(
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/MultilineIfModifier: Favor a normal if-statement over a modifier clause in a multiline statement.